test/dlc_test.json

```diff
@@ -0,0 +1,329 @@
+[ {
+  "testInputs" : {
+    "params" : {
+      "oracleInfo" : {
+        "publicKey" : "a78bf5ed2841a93d4dcc1b719ebbcc90c7648746e5bd55fe7806494a93167c39",
+        "nonce" : "d2d6cb7049ef4edc2d98aa3e235e8d542573cf0c9a735516b9936aba52ae693e"
+      },
+      "contractInfo" : {
+        "8873e27fd5804a500792d3b197ec83e3d13311e161c431b223657b64ec78e355" : 0,
+        "9fc2710e301f8700a8a4c6933366c52045f23a59c92eaf17b0c3d6413861d068" : 200000000
+      },
+      "contractMaturityBound" : 100,
```

Comment:
What is this used for?

Reply:
This is the locktime on the CET

Reply:
nit but maybe better to just call it cetLockTime then (same for next one refundLockTime), it's more explicit IMHO

Reply:
I'm going to make a separate very small PR for this once this gets in (to avoid merge conflicts as this will change all the files)